### PR TITLE
02.25.16

### PR DESCRIPTION
**Pull request by manitghogar**

### CODE REVIEW

File: 31/ex31.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+	if activity == "1":
+		print "Nice choice"
+	elif activity == "2":
+		print "daredevil we got over here"
+	else:
+		print "boring"
+
+else:
+	print "%r is not a valid option" % option
```
Comment (arjunblj) [inline 31/ex31.py:86]: Yo, can you fix the formatting in your Sublime settings? You're using tabs which show up pretty wide. Here's how you fix this—go to Sublime preferences (Hit `Cmd` and `/` at the same time), should open up a file called `Preferences.sublime-settings`. Add the following:

``` json
{
    "auto_complete_commit_on_tab": true,
    "caret_extra_width": 1,
    "caret_style": "phase",
    "draw_minimap_border": true,
    "ensure_newline_at_eof_on_save": true,
    "highlight_line": true,
    "highlight_modified_tabs": true,
    "ignored_packages":
    [
        "Vintage"
    ],
    "line_padding_bottom": 2.5,
    "rulers":
    [
        80
    ],
    "scroll_past_end": true,
    "sidebar_small": true,
    "tab_completion": true,
    "tab_size": 4,
    "translate_tabs_to_spaces": true,
    "trim_trailing_white_space_on_save": true,
    "word_wrap": "auto"
}
```

This should give you some standard white-space settings and help ensure that the style of your code is consistent.

File: 31/ex31.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+	print "3. Go to Thailand"
+
+	vacation = raw_input()
+
+	if vacation == "1" or vacation == "2":
+		print "Have fun, great country"
+	else:
+		print "Good luck in Bangkok"
+
+elif option == "2":
```
Comment (arjunblj) [inline 31/ex31.py:71]: Don't have blank lines between different branches of your if statement, though you can between different statements. e.g. this is OK:

``` python
if a:
   return True
elif b:
   return "hi"
else:
  return False
```

And so is this:

``` python
if a:
  return True
else:
  return False
```

Don't do this:

``` python
if a:
  return True


elif b:
  return False
```

Though if they're separate statements, like below, you're good:

``` python
if a:
  return True
if b:
  return False
```

File: 31/ex31.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+	print "2. Go sky diving"
+
+	activity = raw_input("> ")
+
+	if activity == "1":
+		print "Nice choice"
+	elif activity == "2":
+		print "daredevil we got over here"
+	else:
+		print "boring"
```
Comment (arjunblj) [inline 31/ex31.py:83]: This is a space where your line spacing is correct.

File: 30/ex30.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+if cars > people and trucks > cars:
+	print "We should take the cars, as there are too many trucks"
+
+elif cars < people and trucks < cars: 
+	print "We should not take the cars, let's just take the trucks"
+
+elif people >= trucks:
+	print "Let's just take the trucks"
+
+else:
```
Comment (arjunblj) [inline 30/ex30.py:41]: Fix the spacing here.

File: 26/ex26.py [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+    word = words.pop(0)
+    print word
+
+def print_last_word(words):
+    """Prints the last word after popping it off."""
+    word = words.pop(-1)
+    print word
+
+def sort_sentence(sentence):
+    """Takes in a full sentence and returns the sorted words."""
```
Comment (arjunblj) [inline 26/ex26.py:21]: If you have a docstring (a comment for a function) that's one line then you should format the end of the string on a separate line, e.g.

``` python
def hello():
    """Say hello.
    """
    return "Hello!"
```

File: 31/ex31.py (hunk 1 above)
Comment (manitghogar) [inline 31/ex31.py:86]: Okay done, I think the shortcut is Cmd and ,

**Pull request merged**